Sorting/CountSort.php

```diff
@@ -1,33 +1,39 @@
 <?php
 
-
 /**
- * @param $array
- * @param $min
- * @param $max
- * @return mixed
+ * Sorts an array of integers using the Counting Sort algorithm.
+ *
+ * @param array $array The input array to be sorted.
+ * @param int $min The minimum value in the input array.
+ * @param int $max The maximum value in the input array.
+ * @return array The sorted array.
  */
-function countSort($array, $min, $max)
+function countSort( $array, $min, $max )
```

Comment:
We've tried to stick with the PSR-12 style guide.  Please refer to the guide [here](https://xoops.gitbook.io/xoops-modules-cookbook/coding-standards/standards/styleguide).  These spaces surrounding arguments and parameters aren't necessary

Reply:
This is noted, PSR-12 Standard is followed here.